src/prefer-arrow-functions.ts

```diff
@@ -193,8 +193,8 @@ export default {
     };
 
     const isPrototypeAssignment = (node) => {
-      return context
-        .getAncestors()
+      return sourceCode
```

Comment:
This are the "key" changes, made following the [official documentation](https://eslint.org/docs/latest/use/migrate-to-9.0.0#-removed-multiple-context-methods).